src/e2ir.c

```diff
@@ -55,6 +55,8 @@ unsigned totym(Type *tx);
 Symbol *toSymbol(Dsymbol *s);
 elem *toElem(Expression *e, IRState *irs);
 dt_t **Expression_toDt(Expression *e, dt_t **pdt);
+Symbol *toStringSymbol(const char *str, size_t len, size_t sz);
+Symbol *toStringDarraySymbol(const char *str, size_t len, size_t sz);
```

Comment:
There isn't a header set up for the glue layer stuff, and it doesn't belong in any of the frontend headers.